plugins/web-worker-offloading/third-party/seo-by-rank-math.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Web Worker Offloading integration with Rank Math SEO.
+ *
+ * @since n.e.x.t
+ * @package web-worker-offloading
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Configures WWO for Rank Math SEO and Google Analytics.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @link https://partytown.builder.io/google-tag-manager#forward-events
+ *
+ * @param array<string, mixed>|mixed $configuration Configuration.
+ * @return array<string, mixed> Configuration.
+ */
+function plwwo_rank_math_configure( $configuration ): array {
+	$configuration = (array) $configuration;
+
+	$configuration['globalFns'][] = 'gtag'; // Because gtag() is defined in one script and called in another.
+	$configuration['forward'][]   = 'dataLayer.push'; // Because the Partytown integration has this in its example config.
```

Comment:
ok! it would be good to understand why.

Reply:
Good point. How about:

```suggestion
	$configuration['forward'][]   = 'dataLayer.push'; // See <https://partytown.builder.io/forwarding-event>.
```